docs/project-development-environment.md

```diff
@@ -2,7 +2,19 @@
 
 ## Local prerequisites
 
-Docker and docker-compose is required to be installed.
+| dependency | supported versions | Installation recommendation for Ubuntu 14.04 |
+| ---------- | ------------------ | ---------------------------------------------|
+| <a name='dependency_docker'>docker</a> | 1.10 (1.9 may still work, too) | Follow [the official Docker installation instructions for Linux](https://docs.docker.com/linux/step_one/) |
```

Comment:
To get more space you could put the version in the same field as the name but on the next row.